.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
I add a .gitignore

Reply:
That was removed by mistake, thank you for adding it.